src/table/cells/selections.jsx

```diff
@@ -17,17 +17,22 @@ const useStyles = makeStyles({
 
 export function withSelections(CellComponent) {
   const HOC = (props) => {
-    const { cell, selState, selDispatch } = props;
+    const { cell, selState, selDispatch, className } = props;
     const classes = useStyles();
     const handleMouseUp = (evt) => cell.isDim && selectCell(cell, selState, selDispatch, evt);
     const selectionClass = getSelectionClass(cell, selState);
 
     return (
-      <CellComponent {...props} className={`${selectionClass} ${classes[selectionClass]}`} onMouseUp={handleMouseUp} />
+      <CellComponent
+        {...props}
+        className={`${className} ${selectionClass} ${classes[selectionClass]}`}
```

Comment:
This is not applied to measures, i.e. cells that is not rendered with `withSelections`. We need a new HOC called `withStyling` (wrapping `withSelections` that applies styling independent of selections. Still have to do what you do in here: reset `className` with the new selection classes.

Reply:
Correction, it is applied to meassures, but it still should work without `withSelections`